index.js

```diff
@@ -12,44 +12,83 @@ ValueError.prototype.name = 'ValueError';
 exports.ValueError = ValueError;
 
 
-function def(type, coerce) {
+var unsetVariableError = function(name) {
```

Comment:
i'd find this a bit easier to read if it were `createUnsetVariableError`, but that's a matter of preference.